src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesV2.scala

```diff
@@ -164,6 +182,31 @@ trait UserRoutesV2 extends SamUserDirectives with SamRequestContextDirectives {
       }
     }
 
+  private def getSamUserCombinedState(samUser: SamUser, samRequestContext: SamRequestContext): Route =
+    get {
+      complete {
+        val combinedStateResponse = for {
+          allowances <- userService.getUserAllowances(samUser, samRequestContext)
+          maybeAttributes <- userService.getUserAttributes(samUser.id, samRequestContext)
+          termsOfServiceDetails <- tosService.getTermsOfServiceDetailsForUser(samUser.id, samRequestContext)
+          enterpriseFeatures <- resourceService
+            .listResourcesFlat(
+              samUser.id,
+              Set(ResourceTypeName("enterprise-feature")),
+              Set.empty,
+              Set(ResourceRoleName("user")),
+              Set.empty,
+              includePublic = false,
+              samRequestContext
+            )
```

Comment:
I matched this with [the call that the ui is doing](https://github.com/DataBiosphere/terra-ui/blob/c7623a8574750ca830bd8d48e7fa0c2a415a938a/src/libs/ajax/User.ts#L198)